modules/module/server-ts/ServerModule.ts

```diff
@@ -40,29 +55,24 @@ export interface ServerModuleShape extends CommonModuleShape {
   beforeware?: MiddlewareFunc[];
   // A list of functions to register normal-priority middlewares
   middleware?: MiddlewareFunc[];
```

Comment:
Wouldn't be better to separate app-level middlewares from route-level ones?

Reply:
Add middleware property for apiRouteParams